pkg/pipeline/sink/file.go

```diff
@@ -44,10 +46,20 @@ func (s *FileSink) Start() error {
 }
 
 func (s *FileSink) Close() error {
+	var labels prometheus.Labels
```

Comment:
would be cleaner to pass a stats object to `Uploader` instantiation, and stats are recorded within `remoteUploader.Upload`

Reply:
Thanks for the review and the suggestion, @frostbyte73 
I have refactored the code as you suggested.